optimize_versioned_docs/version-3.10.0/self-managed/optimize-deployment/migration-update/3.9-to-3.10.md

```diff
@@ -26,6 +26,15 @@ configuration options have therefore been removed:
 - ui.header.pathToLogoIcon
 - ui.header.backgroundColor
 
+## Helm chart
+
+For Optimize 3.10.1, a new environment variable introduced redirection URL. However, the change is not compatible with Camunda Helm charts until it is fixed in 3.10.3 (and Helm chart 8.2.9). Therefore, those versions are coupled to certain Camunda Helm chart versions:
```

Comment:
```suggestion
For Optimize 3.10.1, a new environment variable introduced specifying a redirection URL. However, the change is not compatible with Camunda Helm charts until it is fixed in 3.10.3 (and Helm chart 8.2.9). Therefore, those Optimize versions are coupled to certain Camunda Helm chart versions:
```
Don't know about the first sentence could also be `...introduced redirection URL.s` or similar but sounds odd as is.

Reply:
Hmm yeah. I copied from the next docs but you are right - I will update both